README.md

```diff
@@ -275,6 +275,93 @@ Installation instructions assets-iac-terraform-azure by running:
     }
 ```
 
+### K8s Module
```

Comment:
Should we rename it from `K8s` to `AKS`? Is it the official administered Kubernetes Service from Azure